### PR TITLE
receieve message

### PR DESCRIPTION
**Pull request by ahendy**
I changed it to app.run but feel free to put back to jukebot

### CODE REVIEW

File: jukebot/jukebot.py [hunk 1]
```diff
@@ -1,6 +1,38 @@
...
+
+@app.route('/', methods=['GET', 'POST'])
+def listen():
+    '''
+    NOTE request.args returns a multilist AKA a dict {k: []} where 
+    values are lists
+    '''
+    return (
+        verify_token(request) if request.method=='GET'
+                              else receive_message(request))
```
Comment (ben-cunningham) [inline jukebot/jukebot.py:17]: Why not just have this as

```
if request.method == 'GET':
   return verify_token()
else:
   return receive_message(request)
```

seems a little more readable to me

File: jukebot/config/info.github.py [hunk 2]
```diff
@@ -0,0 +1 @@
+TOKEN = '<DEFINED IN token generation in app page>'
```
Comment (ben-cunningham) [inline jukebot/config/info.github.py:1]: What does this mean? Can we just set the token in an environment variable?

File: jukebot/jukebot.py [hunk 3]
```diff
@@ -1,6 +1,38 @@
...
+token = TOKEN
+app = Flask(__name__)
+bot = Bot(TOKEN)
+
+@app.route('/', methods=['GET', 'POST'])
+def listen():
+    '''
+    NOTE request.args returns a multilist AKA a dict {k: []} where 
+    values are lists
+    '''
```
Comment (ben-cunningham) [inline jukebot/jukebot.py]: Let's use double quotes for doc strings for consistency

**Comment by ben-cunningham** [comment]
Those are some great commit messages

File: jukebot/jukebot.py (hunk 1 above)
Comment (ahendy) [inline jukebot/jukebot.py:17]: cuz 1 liner ternary if statement is best statement

File: jukebot/config/info.github.py (hunk 2 above)
Comment (ahendy) [inline jukebot/config/info.github.py:1]: i have a file named info.py with the generated token. yes u could also set an environment variable but i just put info.py into gitignore

File: jukebot/jukebot.py (hunk 3 above)
Comment (ahendy) [inline jukebot/jukebot.py]: can change it then and merge

File: jukebot/config/info.github.py (hunk 2 above)
Comment (ben-cunningham) [inline jukebot/config/info.github.py:1]: @ahendy0 Should we set an environment variable instead? I feel like that would be the best way to do it

File: jukebot/jukebot.py [hunk 4]
```diff
@@ -1,6 +1,38 @@
...
+from flask import Flask, request
+from fbmsgbot.bot import Bot
+from config.info import TOKEN
+
+token = TOKEN
+app = Flask(__name__)
+bot = Bot(TOKEN)
+
+@app.route('/', methods=['GET', 'POST'])
+def listen():
```
Comment (ben-cunningham) [inline jukebot/jukebot.py:10]: Can we name this `callback`?

Comment (ahendy) [inline jukebot/jukebot.py:10]: sure

**Comment by ben-cunningham** [comment]
@ahendy0 Fixed this 4 u

**Pull request merged**